.github/workflows/build-and-test-snap.yml

```
@@ -31,7 +57,7 @@ jobs:
         env:
           MOCK_GPIO: true
           SKIP_TEARDOWN_REMOVAL: true
-          LOCAL_SERVICE_SNAP: ../${{ steps.snapcraft.outputs.snap }}
+          LOCAL_SERVICE_SNAP: ../matter-pi-gpio-commander_${{ github.run_number}}_arm64.snap
```

I believe the snap is in the current directory, as in #51. 

Also, how about using a wildcard to avoid repeating the name?
```suggestion
          LOCAL_SERVICE_SNAP: "*.snap"
```

---

Wrong. Please ignore my comment.